README.md

```diff
@@ -64,21 +64,21 @@ Number of samples: 20
 | HTTP application | 420ms | 330ms | 310ms | 310ms |
 | Laravel          |       |       | 920ms |       |
 
-### Bref 2.x (PHP 8.1)
+### Bref 2.x (PHP 8.2)
 
 | Memory           |   128 |   512 |  1024 |  1769 |
 |------------------|------:|------:|------:|------:|
-| PHP function     | 465ms | 235ms | 210ms | 205ms |
-| HTTP application | 370ms | 280ms | 266ms | 266ms |
+| PHP function     | 445ms | 250ms | 220ms | 215ms |
+| HTTP application | 460ms | 330ms | 310ms | 310ms |
```

Comment:
Got worse, but still better than v1.

I suspect this is because the previous layers (not compiled) had less system libraries. We could double check the content of the layers to trim them more if possible, but again it's still an improvement over v1.